README.md

```diff
@@ -1,29 +1,55 @@
-# Compliance Workshop Setup
+tf-compliance-workshop is a provisioner for AWS that uses Terraform for compliance workshops. It spins up one Automate server and the requisite number of workstations using the playing card system. It assumes you have terraform installed, your AWS ssh keys and secret keys, AWS CLI setup, etc. 
```

Comment:
The word "provisioner" means something very specific in Terraform-world. I might change the first sentence to:

"tf-compliance-workshop" is a Terraform configuration that provisions and configures an environment for compliance workshops.

... I'd also delete the last sentence since you enumerate what to do later in the document, and add "install Terraform on your workstation" as the first item in the Quickstart section.